assets/src/scss/base/_colors.scss

```diff
@@ -1,73 +1,4 @@
-$white: #fff;
```

Comment:
We will need only the social media colors. The rest of the colors are being removed or replaced.

Reply:
Maybe we should remove this file `_social-media-colors.scss` or something like that then? Not a blocker, just a suggestion 🙂